packages/botonic-plugin-flow-builder/tests/helpers/utils.ts

```diff
@@ -97,8 +101,9 @@ export function createRequest({
     plugins,
     getUserCountry: () => user.country,
     getUserLocale: () => user.locale,
-    getSystemLocale: () => user.system_locale,
+    getSystemLocale: () => user.systemLocale,
     setUserCountry: (_country: string) => {
+      user.country = _country
```

Comment:
extra, we can add validations to be more robust by having a regex for COUNTRY_CODE. Maybe not a big thing, but we must be sure that if someone defines 'gb' as a country code, either throw error or make it work by converting. I would directly only allow countries matching UPPER_CASE and two letters